### ISSUE TITLE
Systemback erases my /etc/resolv.conf settings

### ISSUE THREAD

**Issue by fredbird67** [BluewhaleRobot/systemback#42]
I am trying to create a family-friendly Linux distribution that blocks out pornographic websites.  I was able to get my test installation up and running, both live and installed -- only to find that the DNS settings I had in the system were erased when creating the ISO file.  Can you please correct this, or at least add settings to have Systemback not touch settings a user requests be left alone when creating the ISO file?

**Comment by randoms** [comment]
/etc/resolv.conf is managed by systemd-resolved. you should not edit this file. systemback didn't change it.

**Comment by fredbird67** [comment]
Yeah, I eventually figured that out -- my apologies!  In fact, I figured out it was Ventoy doing that, which I have on a flash drive.  After looking for answers on this and not finding any, I came up with an idea to combat that:I copied /etc/resolv.conf to another file I called "last-filter-used.conf", also in /etc.  I then created a short Bash script,  called "load-filter" (in /usr/sbin) that copies last-filter-used to resolv.conf, and then set a cron job as root to automatically run load-filter at bootup, and voila -- bulletproof web filtering in a family-friendly distribution that filters out p0rn that the Lord laid on my heart to create.Yeah, there's Ubuntu Christian Edition -- but I didn't like how it was based on Ubuntu and all the problems that Snap has created (I disabled Snaps in my project, in fact, just like Linux Mint does), plus I didn't like how it was only available with GNOME for a desktop.  Mine, OTOH, is based on Debian Stable and currently comes with your choice of Xfce or Openbox, with plans to eventually create releases with MATE, Cinnamon, and KDE.Again, my apologies!FredSent via the Samsung Galaxy S10, an AT&T 5G Evolution capable smartphone
-------- Original message --------From: randoms ***@***.***> Date: 3/21/24  12:26 AM  (GMT-06:00) To: BluewhaleRobot/systemback ***@***.***> Cc: fredbird67 ***@***.***>, Author ***@***.***> Subject: Re: [BluewhaleRobot/systemback] Systemback erases my /etc/resolv.conf settings (Issue #42) 
/etc/resolv.conf is managed by systemd-resolved. you should not edit this file. systemback didn't change it.
